src/vs/workbench/contrib/mcp/common/mcpServer.ts

```diff
@@ -472,6 +476,8 @@ export class McpServer extends Disposable implements IMcpServer {
 				supportsPrompts: !!handler.capabilities.prompts,
 				supportsResources: !!handler.capabilities.resources,
 				toolCount: tools.length,
+				serverName: handler.serverInfo.name,
```

Comment:
test

Reply:
Can you clarify please. Do you mean there are existing tests that should adapt or fix?

Reply:
sorry no this was a comment testing the GHPR extension 😛

Reply:
Works!